### PR TITLE
[17.0][MIG] stock_picking_filter_lot: Migration to 17.0

### PR DESCRIPTION
**Pull request by fd-oerp**

### CODE REVIEW

**Comment by fd-oerp** [comment]
Hi @LoisRForgeFlow @SimoRubi 

Could you please review this PR and let me know if any corrections are needed?

Thank you!

File: stock_picking_filter_lot/i18n/es.po
Comment (LoisRForgeFlow) [inline stock_picking_filter_lot/i18n/es.po:1]: Thanks for your contribution!

Any specific reason to remove some translation files?

Comment (fd-oerp) [reply stock_picking_filter_lot/i18n/es.po:1]: Hi @LoisRForgeFlow 

Ahh..It's my bad, I added those files again with 17.0 changes. You can review those, please!

Thank you for your reply.

File: stock_picking_filter_lot/views/stock_scrap_view.xml [hunk 1]
```diff
@@ -5,33 +5,13 @@
...
-        <field name="model">stock.scrap</field>
-        <field name="inherit_id" ref="stock.stock_scrap_form_view2" />
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='location_id']" position="attributes">
-                <attribute
-                    name="groups"
-                >stock.group_stock_multi_locations,stock.group_production_lot</attribute>
-            </xpath>
+            <field name="origin" position="after">
+                <field name="location_id" invisible="1" />
```
Comment (LoisRForgeFlow) [inline stock_picking_filter_lot/views/stock_scrap_view.xml]: and this? why making location invisible?

Comment (fd-oerp) [reply stock_picking_filter_lot/views/stock_scrap_view.xml]: Hi @LoisRForgeFlow 

Thank you for your reply!
I updated this view as per it should be, this happned as we are using this module in our project and it had some our customization. My mistake!
You can review pleas.

Thank you

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks! LGTM

**Comment by fd-oerp** [comment]
Hi @pedrobaeza @rousseldenis 

Could you please review this PR and let me know if any updates are needed, or proceed accordingly?

Thank you

**Comment by pedrobaeza** [comment]
Sorry, I'm not using this module.

**Comment by fd-oerp** [comment]
Hi @rousseldenis 

Could you please provide the required input to process this PR? 
Please let me know if anything needs to be fixed or updated from my end.

Thank you

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Pull request merged**